commands/droplets_test.go

```diff
@@ -160,7 +160,9 @@ func TestDropletDelete(t *testing.T) {
 
 func TestDropletDeleteByTag(t *testing.T) {
 	withTestClient(t, func(config *CmdConfig, tm *tcMocks) {
-		tm.droplets.On("DeleteByTag", "my-tag").Return(nil)
```

Comment:
What is the reason behind this change? This function is made to test DeleteByTag and I think it should stay as it is, but if there is good reason behind it, it can be changed.